src/reanimated2/commonTypes.ts

```diff
@@ -94,30 +94,92 @@ export type MapperRegistry = {
   stop: (mapperID: number) => void;
 };
 
-export type Context = Record<string, unknown>;
+export type WorkletClosure = Record<string, unknown>;
 
-export interface WorkletFunction {
-  _closure?: Context;
+interface ReleaseInitData {
+  code: string;
+}
+
+interface DevInitData {
+  code: string;
+  location: string;
+  sourceMap: string;
+  version: string;
+}
+interface ReleaseWorkletBase {
+  _closure: WorkletClosure;
```

Comment:
This single underscore here makes me furious, can we rename it to `__closure`?

Reply:
maybe

Reply:
definitely